src/pages/mf-prod-sheet/Forms/ProductionSheetForm.js

```diff
@@ -0,0 +1,307 @@
+      invalidate()
+    }
+  })
+
+  const editMode = !!formik.values.recordId
+
+  const columns = [
+    {
+      component: 'resourcelookup',
+      label: labels.sku,
+      name: 'sku',
+      props: {
+        endpointId: InventoryRepository.Item.snapshot,
+        valueField: 'recordId',
+        displayField: 'sku',
+        mandatory: true,
+        displayFieldWidth: 2,
+        mapping: [
+          { from: 'recordId', to: 'itemId' },
+          { from: 'sku', to: 'sku' },
+          { from: 'name', to: 'itemName' }
+        ],
+        columnsInDropDown: [
+          { key: 'sku', value: 'SKU' },
+          { key: 'name', value: 'Name' }
+        ]
+      }
+    },
+    {
+      component: 'textfield',
+      label: labels.itemName,
+      name: 'itemName',
+      props: {
+        readOnly: true
+      }
+    },
+    {
+      component: 'numberfield',
+      label: labels.qty,
+      name: 'qty'
+    },
+    {
+      component: 'numberfield',
+      label: labels.orderedQty,
+      name: 'orderedQty',
+      props: {
+        readOnly: true
+      }
+    },
+    {
+      component: 'textfield',
+      label: labels.notes,
+      name: 'notes'
+    },
+  ]
+
+  async function getData(recordId) {
+    return await getRequest({
+      extension: ManufacturingRepository.ProductionSheet.get,
+      parameters: `_recordId=${recordId}`
+    })
+  }
+
+  async function getDataGrid() {
+    return await getRequest({
+      extension: ManufacturingRepository.ProductionSheetItem.qry,
+      parameters: `_psId=${recordId}&_functionId=${SystemFunction.ProductionSheet}`
+    })
+  }
+
+  useEffect(() => {
+    ;(async function () {
+      if (recordId) {
+        const res = await getData(recordId)
+
+        const res2 = await getDataGrid()
+
+        formik.setValues({
+          ...res.record,
+          items: res2.list.map(item => ({
+            ...item,
+            id: item.seqNo,
+            orderedQty: item.orderedQty ?? 0
+          })),
+          date: !!res?.record?.date ? formatDateFromApi(res?.record?.date) : null
+        })
+      }
+    })()
+  }, [])
+
+  return (
+    <FormShell
+      resourceId={ResourceIds.ProductionSheet}
+      form={formik}
+      maxAccess={maxAccess}
+      editMode={editMode}
+      functionId={SystemFunction.ProductionSheet}
+    >
+      <VertLayout>
+        <Grow>
+          <Grid container spacing={4}>
+            <Grid item xs={6}>
+              <ResourceComboBox
+                endpointId={SystemRepository.DocumentType.qry}
+                parameters={`_dgId=${SystemFunction.ProductionSheet}&_startAt=${0}&_pageSize=${50}`}
+                filter={!editMode ? item => item.activeStatus === 1 : undefined}
+                name='dtId'
+                label={labels.documentType}
+                readOnly={editMode}
+                valueField='recordId'
+                displayField='name'
+                required
+                values={formik.values}
```

Comment:
![image](https://github.com/user-attachments/assets/208e38dd-2245-47a4-81a1-4a7e33818509)
document type and site required but there is no error if they are empty